web-marketplace/src/clients/regen/Regen.Routes.tsx

```diff
@@ -141,6 +142,19 @@ export default function RegenRoutes({ reactQueryClient }: RouterProps) {
         rpcQueryClient,
         address: activeWalletAddr,
         languageCode: selectedLanguage,
+        // Sets React Router’s basename to the current locale (e.g., /en, /es)
+        // so all legacy React Router paths (/, /projects, etc.) resolve under /{lang}.
+        // TODO: Remove `basename` when the legacy React Router is fully migrated
```

Comment:
well I guess this entire file will be removed once everything is migrated to the app router, right?